packages/palette/src/uiColors.ts

```diff
@@ -0,0 +1,54 @@
+export const white = '#FFFFFF';
+export const black = '#061621';
+
+interface HueRange {
+  dark3: string;
+  dark2: string;
+  dark1?: string;
+  base: string;
+  light1?: string;
+  light2: string;
+  light3: string;
+}
+
+export const gray: HueRange = {
+  dark3: '#21313C',
```

Comment:
Alternative API:

`uiColors.dark[0]` so it's actually an array of values. 

I don't actually like it more than what you have but wanted to brainstorm aloud.

Reply:
These names directly map to the color palette framework that we built that designers will also be using. Keeping these names will make sure nothing is lost in communication.